src/AppBundle/Entity/Post.php

```diff
@@ -95,7 +95,7 @@ class Post
      *      mappedBy="post",
      *      orphanRemoval=true
      * )
-     * @ORM\OrderBy({"publishedAt" = "DESC"})
+     * @ORM\OrderBy({"publishedAt": "DESC"})
```

Comment:
The new syntax could confuse Newbies while they are writing their first Symfony app. The annotations from the framework bundle are still using `=` and they might wonder, why the annotations in this case look complete different. They might not recognize the difference of the two seperate projects.